app/views/items/index.haml

```diff
@@ -10,10 +14,10 @@
           .dropdown
           %button#dropdownMenuButton.btn.btn-secondary.my-2.dropdown-toggle{"aria-expanded" => "false", "aria-haspopup" => "true", "data-toggle" => "dropdown", :type => "button"} Sort By:
           .dropdown-menu{"aria-labelledby" => "dropdownMenuButton"}
-            %a.dropdown-item{:href => "#"} Price (Low - High)
-            %a.dropdown-item{:href => "#"} Price (High - Low)
-            %a.dropdown-item{:href => "#"} Manufacturer (A-Z)
-            %a.dropdown-item{:href => "#"} Manufacturer (Z-A)
+            %a.dropdown-item{:href => items_path} Price (Low - High)
```

Comment:
you could make these actually work by passing a hash to items path:
`items_path(sort: 'price', order: 'desc')` which will produce a path like `items?sort=price&order=desc` which are url params.  Rails will make those available to you as `params['sort']`